### DISCUSSION TITLE
Welcome to MSEdgeTweaker Discussions!

### DISCUSSION THREAD

**Discussion by TheBobPony** [TheBobPony/MSEdgeTweaker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share any ideas that you have!
  * Engage with other community members.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SRC267** [comment]
Great utility, one thing I would like to see added is way to disable the bing logo element from the search bar ![bing.png](https://github.com/TheBobPony/MSEdgeTweaker/assets/140402431/75a6dfa2-46f2-422b-89e0-0150263aee5e)

I can't see an option for that, unless I missed it somewhere deep in the options.

**Reply by SmearODeer** [reply]
Using Group Policy Editor
	Open Group Policy Editor: Press Win + R, type gpedit.msc, and press Enter.
	Navigate to Edge Settings: Go to Computer Configuration > Administrative Templates > Microsoft Edge. 
        Disable Copilot: Look for a setting named Allow Edge Copilot (Side Pane) or similar. Double-click it and set it to Disabled.
	Apply the Changes: Click Apply and then OK.

Using Registry Editor
	Open Registry Editor: Press Win + R, type regedit, and press Enter. 
	Navigate to the Edge Key: Go to HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge.
	Create a New DWORD Value: Right-click on Edge, select New > DWORD (32-bit) Value, and name it EdgeCopilotEnabled.
	Set the Value: Double-click the new value and set it to 0.
	Restart Edge: Close and reopen Microsoft Edge to apply the changes.

Using Edge Flags (if applicable)
	Open Edge Flags: In Edge, type edge://flags in the address bar and press Enter.
	Search for Copilot: In the search bar, type Copilot or Bing.
	Disable the Flag: If there is a relevant flag, set it to Disabled.
	Restart Edge: Close and reopen Microsoft Edge to apply the changes.

Using Command Line (for advanced users)
	Open Command Prompt as Administrator: Press Win + X, select Command Prompt (Admin) or Windows PowerShell (Admin).
	Execute the Command: Enter the following command to disable the feature:
		reg add "HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge" /v EdgeCopilotEnabled /t REG_DWORD /d 0 /f
	Restart Edge: Close and reopen Microsoft Edge to apply the changes.
